### PR TITLE
Updating HttpKernel.php : Adding Add PHPDoc block to varToString method

### PR DESCRIPTION
**Pull request by shreypuranik**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      |  no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes  
| License       | MIT

This update adds PHPDoc Block to the varToString method in HttpKernel.php

### CODE REVIEW

File: src/Symfony/Component/HttpKernel/HttpKernel.php [hunk 1]
```diff
@@ -250,6 +250,13 @@ private function handleException(\Exception $e, Request $request, int $type): Re
         }
     }
 
+    /*
+     * Returns a more explicit  definition of the supplied variable
```
Comment (ro0NL) [inline src/Symfony/Component/HttpKernel/HttpKernel.php]: `Returns an exported string for the specified variable.`

File: src/Symfony/Component/HttpKernel/HttpKernel.php [hunk 2]
```diff
@@ -250,6 +250,13 @@ private function handleException(\Exception $e, Request $request, int $type): Re
         }
     }
 
+    /*
+     * Returns a more explicit  definition of the supplied variable 
+     * as a string
+     *
+     * @param $var
```
Comment (ro0NL) [inline src/Symfony/Component/HttpKernel/HttpKernel.php]: redundant

File: src/Symfony/Component/HttpKernel/HttpKernel.php [hunk 3]
```diff
@@ -250,6 +250,13 @@ private function handleException(\Exception $e, Request $request, int $type): Re
...
+     * @return string
```
Comment (ro0NL) [inline src/Symfony/Component/HttpKernel/HttpKernel.php]: also redundant

File: src/Symfony/Component/HttpKernel/HttpKernel.php (hunk 1 above)
Comment (ro0NL) [reply src/Symfony/Component/HttpKernel/HttpKernel.php]: actually: `Returns a human-readable string for the specified variable.`

**Review by ro0NL — COMMENTED** [review]
i suggest to target 3.4

File: src/Symfony/Component/HttpKernel/HttpKernel.php (hunk 1 above)
Comment (shreypuranik) [reply src/Symfony/Component/HttpKernel/HttpKernel.php]: Hi @ro0NL , Thank you for reviewing these updates. I've committed a further update to this Pull Request taking your comments into account. I've also updated the opening comment to reference the 3.4 version. 

Please can you review these updates whenever you have a chance to do so?

**Comment by shreypuranik** [comment]
Closing PR with a view to open a new PR to merge into 3.4 branch as per @ro0NL suggestion.

**Pull request closed without merging**